src/sync/list.rs

```diff
@@ -93,17 +96,21 @@ impl<T> List<T> {
     }
 
     /// Inserts `data` into the head of the list.
+    #[inline]
     pub fn insert<'scope>(&'scope self, data: T, scope: &'scope Scope) -> Ptr<'scope, Node<T>> {
         Self::insert_internal(&self.head, data, scope)
     }
 
-    /// Inserts `data` after `to`.
+    /// Inserts `data` after into the head of the list.
```

Comment:
Should this be "inserts `data` after `after` into the list."?

Reply:
Thanks!  I just pushed a new commit that changes the doc as you said.